src/components/DevelopmentGuide/V3MigrationGuide.mdx

```diff
@@ -0,0 +1,163 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Button from "../Button/Button";
+import Link from "../Link/Link";
+import Text from "../Text/Text";
+
+<Meta title="Development Guide/v3 Migration Guide" />
+
+# Reservoir v3 Migration Guide
+
+## Table of Contents
+
+- {<Link href="#general-updates-requirements">General Updates & Requirements</Link>}
+- {<Link href="#upgrade-path">Upgrade Path</Link>}
+- {<Link href="#breaking-changes">Breaking Changes</Link>}
+- {<Link href="#additional-testing-needs">Additional Testing Needs</Link>}
+
+## General Updates & Requirements
```

Comment:
What do you think of actually including the command

```
$ npm install @nypl/design-system-react-components@latest
```

Reply:
Oh that's a good point. Hmm I think that command will make it include the `^` in package.json which can lead to issues. I'll be explicit about it in this next update.

Reply:
Love it!